app/api/domains/osu.py

```diff
@@ -1667,6 +1667,18 @@ async def register_account(
     # are safe for registration.
     errors: Mapping[str, list[str]] = defaultdict(list)
 
+    # Disable in-game registration if enabled
+    if app.settings.DISABLE_INGAME_REGISTRATION:
+        errors["password"].append(
+            "The in-game registration is disabled. Please register on the website.",
```

Comment:
```suggestion
            "In-game registration is disabled. Please register on the website.",
```

`The` doesn't really have a place here